packages/atlas/src/providers/videoWorkspace/hooks.ts

```diff
@@ -84,11 +84,11 @@ export const useVideoWorkspaceData = () => {
     : {
         video: {
           id: video?.media?.id ?? null,
-          url: video?.media?.resolvedUrl,
+          url: video?.media?.resolvedUrls[0],
```

Comment:
I'm not sure about the context of this file, but shouldn't we use asset resolution here? Ie use `getSingleAssetUrl` function?